docs/articles_en/documentation/openvino_ir/operation_sets/operations_specifications/detection/PriorBox_8.rst

```diff
@@ -160,7 +160,7 @@ PriorBox
 
 * *min_max_aspect_ratios_order*
 
-  * **Description**: *min_max_aspect_ratios_order* is a flag that denotes the order of output prior box. If set true, the output prior box is in [min, max, aspect_ratios] order, which is consistent with Caffe. Note that the order affects the weights order of the preceding convolution layer and does not affect the final detection results.
+  * **Description**: *min_max_aspect_ratios_order* is a flag that denotes the order of output prior box. If set true, the output prior box is in [min, max, aspect_ratios] order. Note that the order affects the weights order of the preceding convolution layer and does not affect the final detection results.
```

Comment:
This relates to OpenVINO supporting the framework indirectly. We could consult it with developers.

Reply:
agreed with Karol

Reply:
Change applied in https://github.com/openvinotoolkit/openvino/pull/22509/commits/dcc29c2247b8ff64a8d80be7591eff02ce87d58f